### PR TITLE
suppress compile error

### PR DESCRIPTION
**Pull request by vvakame**

### CODE REVIEW

File: auth/src/ts/main.ts [hunk 1]
```diff
@@ -14,16 +14,13 @@
 
 // This sample is TypeScript ported version of https://github.com/firebase/quickstart-js.
 
-/// <reference path="../../../node_modules/firebase/app.d.ts" />
-/// <reference path="../../../node_modules/firebase/firebase.d.ts" />
```
Comment (vvakame) [inline auth/src/ts/main.ts:18]: this lines is not required.
tsc can lookup these definition files. see `cat node_modules/firebase/package.json | grep types`.

http://qiita.com/vvakame/items/72d22e33632178f7db24

File: auth/src/ts/main.ts [hunk 2]
```diff
@@ -14,16 +14,13 @@
...
-
 import * as firebase from 'firebase';
 
 const config = {
     apiKey: "<apiKey>",
     authDomain: "<domain>",
     databaseURL: "<database URL>",
-    storageBucket: "<storage bucket>"
+    storageBucket: "<storage bucket>",
```
Comment (vvakame) [inline auth/src/ts/main.ts:23]: as you like.

File: auth/src/ts/main.ts [hunk 3]
```diff
@@ -34,10 +31,10 @@ function toggleSignIn() {
     if (firebase.auth().currentUser) {
         firebase.auth().signOut();
     } else {
-        const email = (<HTMLInputElement>document.getElementById('email')).value;
+        const email = (document.getElementById('email') as HTMLInputElement).value;
```
Comment (vvakame) [inline auth/src/ts/main.ts:34]: as you like.

https://github.com/Microsoft/TypeScript/pull/3564

File: auth/src/ts/main.ts [hunk 4]
```diff
@@ -34,10 +31,10 @@ function toggleSignIn() {
...
         const password = (<HTMLInputElement>document.getElementById('password')).value;
         firebase.auth().createUserWithEmailAndPassword(email, password)
-        .catch((e) => {
+        .catch((e: firebase.FirebaseError) => {
```
Comment (vvakame) [inline auth/src/ts/main.ts:37]: You should make up type annotation manually.

**Comment by ymotongpoo** [comment]
I appreciate your advice.

**Pull request merged**